iDiff/README.md

```diff
@@ -0,0 +1,245 @@
+# iDiff
+
+[![Build
+Status](https://travis-ci.org/GoogleCloudPlatform/runtimes-common.svg?branch=master)](https://travis-ci.org/GoogleCloudPlatform/runtimes-common)
+[![codecov](https://codecov.io/gh/GoogleCloudPlatform/runtimes-common/branch/master/graph/badge.svg)](https://codecov.io/gh/GoogleCloudPlatform/runtimes-common)
+
+## What is iDiff?
+
+iDiff is an image differ command line tool.  iDiff can diff two images along several different criteria, currently including:
+- Docker Image History
+- Image file system
+- apt-get installed packages
+- pip installed packages
+- npm installed packages
+
+This tool can help you as a developer better understand what is changing within your images and better understand what your images contain.
+
+## Installation
+
+### macOS
+```shell
+brew cask install iDiff
+```
+
+### Linux
+```shell
+curl -Lo iDiff https://storage.googleapis.com/iDiff/releases/latest/iDiff-linux-amd64 && chmod +x iDiff && sudo mv iDiff /usr/local/bin/
+```
+
+### Windows
+Download the [iDiff-windows-amd64.exe](https://storage.googleapis.com/iDiff/releases/latest/minikube-windows-amd64.exe) file, rename it to `iDiff.exe` and add it to your path
```

Comment:
don't want to link to minikube here haha. let's build and push these binaries this afternoon and then update these links